tests/test_query_engine.py

```diff
@@ -47,7 +63,7 @@ def test_run_generated_sql_get_single_column_default_population(engine):
     class Cohort(OldCohortWithPopulation):
         output_value = table("clinical_events").first_by("patient_id").get("code")
 
-    assert engine.extract(Cohort) == [dict(patient_id=1, output_value="Code1")]
+    assert engine.extract(convert(Cohort)) == [dict(patient_id=1, output_value="Code1")]
```

Comment:
With this conversion, does that mean we can drop the conversion inside the `QueryEngine` once we've done slashing and burning tests elsewhere that use the old v2 DSL? If so, we should add a note to that story (https://app.shortcut.com/ebm-datalab/story/620/drop-support-for-defining-studies-via-the-cohort-extractor-v2-dsl).

Reply:
Sadly not. In fact, almost the opposite. The thing we need to drop after the slash and burn is having the conversion be _optional_ i.e. we should insist on being supplied with new query model (that behind the scenes we have to convert to old in order to evaluate). But I'll add a note to the story to make that clear.

Reply:
Oh yes, of course. But we can drop something once that's story's done. Thanks.